WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -328,7 +353,7 @@ def do_actions_for_selected
       if params[:registrations_action] == "export-selected"
         format.js { render :redirect_to_export }
       else
-        format.js { render :do_actions_for_selected }
+        format.js { render :do_actions_for_selected, locals: { registration_error_ids: registration_error_ids } }
```

Comment:
To avoid passing locals directly you can just use an instance variable like `@registration_error_ids`.

Reply:
Is that better than passing a local variable? I was actually avoiding creating an instance variable here on purpose :) Hopefully limiting the scope of a variable helps with memory usage?

Reply:
Instance variables is the standard way to pass data to the view, here `.js.erb` is no different from a normal `.html.erb` view. Passing everything manually would be tedious and I don't think there's any benefit to that.

Reply:
Done!